static/js/controllers/inbox.js

```diff
@@ -378,43 +382,30 @@ var feedback = require('../modules/feedback'),
           render: function() {
             $scope.openTourSelect();
             var id = 'org.couchdb.user:' + Session.userCtx().name;
-            UpdateUser(id, { known: true })
-              .catch(function(err) {
-                $log.error('Error updating user', err);
-              });
+            UpdateUser(id, { known: true }).catch(function(err) {
+              $log.error('Error updating user', err);
+            });
```

Comment:
Yeah I think so. I didn't change it so I'm not 100% sure but I think because it's the last one it's ok. I'll think about adding it because it tripped me up too.